packages/studio-be/tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "paths": {
       "botpress/sdk": ["./sdk/botpress.d.ts"]
     },
-    "types": ["reflect-metadata", "jest"]
+    "types": ["reflect-metadata", "jest"],
+    "skipLibCheck": true
```

Comment:
Why ?

Reply:
typescript was trying to build an npm package

Reply:
I removed it and it builds without problem